File: js-pr/56/javascript/code-quality-naming/input_validator_45.js
// {fact rule=code-quality-naming@v1.0 defects=0}
function validateCustomerInput(customerInput) {
    let isInputValid = customerInput !== null && customerInput.length > 0;
    if (!isInputValid) {
        throw new Error("Invalid customer input");
    }
}
// {/fact}

File: python-pr/52/python/code-quality-naming/input_validator_45.py
# {fact rule=code-quality-naming@v1.0 defects=0}
def validate_customer_input(customer_input):
    is_input_valid = customer_input is not None and len(customer_input) > 0
    if not is_input_valid:
        raise ValueError("Invalid customer input")
# {/fact}